### PR TITLE
ExitWithError() - more upgrades from Exit()

### PR DESCRIPTION
**Pull request by edsantiago**
Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

File: test/e2e/run_entrypoint_test.go [hunk 1]
```diff
@@ -17,7 +16,7 @@ CMD []
 		podmanTest.BuildImage(dockerfile, "foobar.com/entrypoint:latest", "false")
 		session := podmanTest.Podman([]string{"run", "foobar.com/entrypoint:latest"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Or(Exit(126), Exit(127)))
+		Expect(session).Should(ExitWithError(126, "open executable: Operation not permitted: OCI permission denied"))
```
Comment (edsantiago) [inline test/e2e/run_entrypoint_test.go:19]: 127 was runc.

**Review by edsantiago — COMMENTED** [review]
This is the last of my `ExitWithError()` PRs. There are still a lot of `Exit(N)` left, but none are of the form that can be handled by `ExitWithError()`

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**